doc/api/animation_api.rst

```diff
@@ -19,7 +19,6 @@ The easiest way to make a live animation in matplotlib is to use one of the
 
 .. autosummary::
    :toctree: _as_gen
-   :template: autosummary_inher.rst
```

Comment:
This seems un-related/un-intended?  The inherited members on the Animation classes help quite a bit.

Reply:
That's from https://github.com/matplotlib/matplotlib/pull/11931, see reasoning there (tldr: the default template already shows inheritance now).
Built this PR on top of it so that I only have to rewrite one template, not two.